tensorflow/core/util/mkl_util.h

```diff
@@ -1983,7 +1983,7 @@ class MklPrimitiveFactory {
   ~MklPrimitiveFactory() {}
 
   MklPrimitive* GetOp(const string& key) {
-#ifndef DNNL_AARCH64_USE_ACL
+#if !defined(DNNL_AARCH64_USE_ACL) || (defined(DNNL_AARCH64_USE_ACL) && !defined(ENABLE_ONEDNN_OPENMP))
```

Comment:
This can be simplified to
```suggestion
#if !defined(DNNL_AARCH64_USE_ACL) || !defined(ENABLE_ONEDNN_OPENMP)
```
Same for other places.
Please also apply code formatter (lines can't be longer than 80 characters).

Reply:
Thanks @penpornk. This has been fixed in 8dae415 and clang-format was run and one file that reported formatting issue was fixed in ecac66d.